src/family.js

```diff
@@ -144,6 +144,11 @@ Family.formatRule_ = function(ruleObj) {
  * Create a column family.
  *
  * @param {object} [options] See {@link Table#createFamily}.
+ * @param {function} callback The callback function.
```

Comment:
These are the response parameters: https://github.com/muraliQlogic/nodejs-bigtable/blob/9049097a93e294576dd3cf0fed639678e1fa7c9a/src/table.js#L197-L199